options_test.go

```diff
@@ -261,3 +261,29 @@ func TestParseURL(t *testing.T) {
 		})
 	}
 }
+
+var hello struct {
+	Test string
+}
+
+func TestOptions_ToURL(t *testing.T) {
```

Comment:
Another good test case might be to try to parse the URL that is output from ToURL to make sure that it is still a valid input.

Reply:
yeah that makes sense to me.. good call